docs/sources/developers/angular_deprecation/angular-plugins.md

```diff
@@ -277,14 +317,6 @@ Latest Version: 1.0.1 | Signature: Community | Last Updated: 2021
 
 > **Migration available - plugin superseded:** this plugin was [discontinued in favour of the InfluxDB data source](https://github.com/ntop/ntopng-grafana-datasource) - a Core plugin included in Grafana, additional guidance is available [here](https://www.ntop.org/guides/ntopng/basic_concepts/timeseries.html#influxdb-driver).
 
-### [Oracle Cloud Infrastructure Logs](https://grafana.com/grafana/plugins/oci-logs-datasource/)
-
-Latest Version: 3.0.0 | Signature: Commercial | Last Updated: 2023
-
-### [Oracle Cloud Infrastructure Metrics](https://grafana.com/grafana/plugins/oci-metrics-datasource/)
-
-Latest Version: 4.0.0 | Signature: Commercial | Last Updated: 2023
-
```

Comment:
It looks like the latest version of OCI log (3.0.0) and OCI metrics (4.0.1) are still using Angular.

It wasn't detected because apparently I forgot to update the CDN (which has to be done manually).
I will take care of updating the files on the CDN and the hardcoded detections, but we may have to come up with a plan to automatically update the CDN at some point.

Those two entries to be reverted. I checked the other plugins (Cognite Data Fusion, OSISoft, Splunk) and they are indeed still using Angular so those are good 👍 

Sorry about this 😅

Reply:
No problem at all - and thanks for checking! Will re-add them

Reply:
@xnyo could you review now with the latest commit 🙏 

also, just to check:
> I checked the other plugins (Cognite Data Fusion, OSISoft, Splunk) and they are indeed still using Angular so those are good

You do mean that they're _not_ using Angular now right?

Reply:
My bad! Yes, I meant that they are not using Angular, they have been migrated to React :)